### ISSUE TITLE
Testing hanging on macos

### ISSUE THREAD

**Issue by qntnrbns** [jupyter/jupyter_server#141]
I am trying to run the tests on my local, and the `nbconvert/tests/test_nbconvert_handlers.py::APITest::test_from_file` test just hangs indefinitely. Is there something that I need to do to get this to work properly? Any advice would be appreciated. 

```bash
$ py.test --cov notebook -v --pyargs jupyter_server
================================================================================= test session starts =================================================================================
platform darwin -- Python 3.7.5, pytest-5.2.2, py-1.8.0, pluggy-0.13.0 -- /anaconda3/envs/jp_server/bin/python
cachedir: .pytest_cache
rootdir: /Users/quinton.robbins/oss/jupyter_server
plugins: nbval-0.9.3, cov-2.8.1
collected 206 items                                                                                                                                                                   

jupyter_server/auth/tests/test_security.py::test_passwd_structure PASSED                                                                                                        [  0%]
jupyter_server/auth/tests/test_security.py::test_roundtrip PASSED                                                                                                               [  0%]
jupyter_server/auth/tests/test_security.py::test_bad PASSED                                                                                                                     [  1%]
jupyter_server/auth/tests/test_security.py::test_passwd_check_unicode PASSED                                                                                                    [  1%]
jupyter_server/extension/tests/test_extension.py::TestExtensionAppInitialize::test_instance_creation PASSED                                                                     [  2%]
jupyter_server/extension/tests/test_extension.py::TestExtensionAppInitialize::test_instance_creation_with_arg PASSED                                                            [  2%]
jupyter_server/extension/tests/test_extension.py::TestExtensionAppInitialize::test_initialize FAILED                                                                            [  3%]
jupyter_server/extension/tests/test_extension.py::TestExtensionServerInitialize::test_initialize_server PASSED                                                                  [  3%]
jupyter_server/extension/tests/test_extension.py::TestExtensionServerInitialize::test_initialize_server_argv FAILED                                                             [  4%]
jupyter_server/extension/tests/test_extension.py::TestExtensionServerInitialize::test_initialize_mixed_argv FAILED                                                              [  4%]
jupyter_server/nbconvert/tests/test_nbconvert_handlers.py::APITest::test_from_file
```

**Comment by Zsailer** [comment]
Thanks for raising this issue.

Fortunately, I'm in the middle of [rewriting the tests completely in pytest](https://github.com/Zsailer/jupyter_server_tests). This problem should go away after this migration (hopefully happening this week). 

See [this issue](https://github.com/jupyter/jupyter_server/issues/136) tracking the pytest migration.

**Comment by Zsailer** [comment]
Should be solved in #152. Closing for now. Feel free to reopen if tests are still stalling.

**Comment by qntnrbns** [comment]
looks good now
